docker/examples/oracle/Dockerfile.sqlplus

```
@@ -0,0 +1,17 @@
+FROM liquibase/liquibase:latest
+
+# Install SQLPlus
+USER root
+RUN apt-get update && apt-get -y install libaio1 libaio-dev && rm -rf /var/lib/apt/lists/*
+RUN mkdir /opt/oracle
+# You must already have the sqlplus archives downloaded from Oracle
+COPY instantclient-sqlplus-linux.x64-19.12.0.0.0dbru.zip instantclient-basic-linux.x64-19.12.0.0.0dbru.zip .
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)